packages/filebrowser-extension/src/index.ts

```diff
@@ -517,12 +517,14 @@ function createContextMenu(model: Contents.IModel | undefined, commands: Command
 /**
  * Create a launcher for a given filebrowser widget.
  */
-function createLauncher(commands: CommandRegistry, browser: FileBrowser): Promise<Launcher> {
+function createLauncher(commands: CommandRegistry, browser: FileBrowser): Promise<MainAreaWidget<Launcher>> {
```

Comment:
What do these changes have to do with tsv support?

Reply:
Nothing really. I did rebase of my branch from master before push. But I don't undrestand why this commit is included?

Reply:
I agree that it's weird. But I really did not touch this file. What should I do? I'm not sure how to revert this change. 

Here is what I did:
```
git checkout master
git pull upstream master
git checkout tsv-file-support
git pull
git rebase master
git push
```

Reply:
It may be that github is showing the diff based on a common ancestor commit, and not taking into account the rebase. Once everything else is cleaned up in this PR, I'll check it out locally and make sure the actual diff is okay.

Reply:
Thank you. Please let me know if there is anything that I can change to fix this.

Reply:
Something does seem odd. For example, 228498d63cc4a3c754506083e57713a3032b0586 in this branch matches up with af09753672f358551ae655a54a5fc30db5fc5e22 in master, but they are actually different commits.

Reply:
Yes it's weird. I think the simplest solution would be to close this pull request and start a new branch from current master.

Reply:
Yes, it looks like you may have rebased a series of commits that had *already* been merged to master. One thing you can try is to do an interactive rebase `git rebase -i master`, which will bring up a text editor showing the commits in this PR. You can then delete the unwanted commits from the branch (with possibly some conflict resolution). But it may be simpler to recreate the changes in a fresh place.

Thanks for the contributions, and working through these annoyances!